example/rater/classification.ipynb

```diff
@@ -133,6 +152,13 @@
     "client = RaterClient(config)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we can run this client. Client will sample output from LLM 3 times to generate final result."
```

Comment:
Let' add a header here named "Run the client". Also, we can add more details such as "Then we can run the client. For each item in the `raw_input`, the Client will generate an explanation and a final label `Yes` or `No`. The label is decided by taking the majority votes from sampling the LLM output 3 times, which improved stability compared with outputting 1 time.